LordUche/ireporter-react
# #164034244 Add Profile Page

#### What does this PR do?
Adds the profile page
#### Description of Task to be completed?
- create a `Profile` component
- create `IncidentList` component
- create `actions` for fetching a user's incidents
#### How should this be manually tested?
- ensure that the build is passing on `Travis CI`
- check the review app and test that everything is okay
#### Any background context you want to provide?
As a User, I should be able to view my profile so I can view all red-flag/intervention incidents reported by me as well as the number of resolved, unresolved and rejected reports
#### What are the relevant pivotal tracker stories?
[#164034244](https://www.pivotaltracker.com/story/show/164034244)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
